crates/sandbox/src/lib.rs

```diff
@@ -91,6 +92,37 @@ pub struct SandboxConfig {
     #[get = "pub"]
     #[builder(default)]
     pinns: Pinns,
+
+    #[get = "pub"]
+    #[builder(default)]
+    sysctls: HashMap<String, String>,
+
+    #[get = "pub"]
+    cgroup_parent: PathBuf,
+
+    #[get = "pub"]
+    #[builder(default)]
+    run_as_user: Option<i64>,
+
+    #[get = "pub"]
+    #[builder(default)]
+    run_as_group: Option<i64>,
+
+    #[get = "pub"]
+    #[builder(default)]
+    supplemental_groups: Vec<i64>,
+
+    #[get = "pub"]
+    #[builder(default)]
+    privileged: bool,
+
+    #[get = "pub"]
+    #[builder(default)]
+    seccomp_profile: Option<String>,
+
+    #[get = "pub"]
+    #[builder(default)]
+    readonly_rootfs: bool,
```

Comment:
Why don't you extract the security-related stuff with the name LinuxSandboxSecurityContext, etc.? I refered cri-o as a model for this name.

Reply:
Cri-o uses the request as is, whereas in our case sandbox and the CRI service are decoupled, but sure we can group security related stuff together.